src/backend/marsha/core/api/video.py

```diff
@@ -98,7 +98,8 @@ def get_permissions(self):
             ]
         elif self.action in ["destroy"]:
             permission_classes = [
-                permissions.IsVideoPlaylistAdmin | permissions.IsVideoOrganizationAdmin
+                permissions.IsObjectPlaylistAdmin
```

Comment:
I think it can be `IsObjectPlaylistAdminOrInstructor`

Reply:
Currently the playlist instructors cannot "create" video, should they be able to delete them ? or should we allow them to create video ? (note: the "create" permission has been added in another PR)

Reply:
then can do both. Create and delete.